frontend/src/app/modules/projects/projects-page/projects-page.component.sass

```diff
@@ -1,40 +1,50 @@
 @import "/src/styles/color"
 @import "/src/styles/weights"
+@import "/src/styles/reusable"
 
-.projects-sector
+.upper-panel-container
     display: flex
-    flex-direction: column
+    justify-content: space-between
+    align-items: center
     margin-left: 79px
-    gap: 16px
-    margin-right: 2%
-    margin-top: 2%
+    padding: 10px
 
-    .upper-panel-container
-        display: flex
-        justify-content: space-between
-        align-items: center
-
-
-    .horizontal-separator
-        border-bottom: 1px solid $light-gray
-        width: 100%
+.horizontal-separator
+    border-bottom: 1px solid $light-gray
+    width: 100%
 
-    .mat-h3
-        padding-left: 20px
-        font-size: 28px
-        font-weight: $bold-weight
+.projects-sector
+    display: flex
+    flex-direction: column
+    height: 100%
+    margin-left: 79px
 
     .card-container
-        margin-top: 30px
+        max-height: 550px
```

Comment:
I think you need a margin top from the cards, because they are very close to the horizontal separator

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-squirrel/assets/118636435/57457125-c591-4379-a067-aa59e875c41e)
And also, is it normal that the lower limit is so big?)